### PR TITLE
ci: test on linux and osx

### PR DESCRIPTION
**Pull request by ChristianMurphy**

### CODE REVIEW

File: packages/create-mdx/readme.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ abide by its terms.
 
 <!-- Definitions -->
 
-[build]: https://travis-ci.org/mdx-js/mdx
+[build]: https://travis-ci.com/mdx-js/mdx
```
Comment (wooorm) [inline packages/create-mdx/readme.md]: Is .com correct? 🤔 [org](https://api.travis-ci.org/mdx-js/mdx.svg?branch=master) works for me, [com](https://travis-ci.com/mdx-js/mdx.svg?branch=master) doesn't?

Comment (ChristianMurphy) [reply packages/create-mdx/readme.md]: Working to get the repo migrated over to the updated Travis CI https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps

Comment (ChristianMurphy) [reply packages/create-mdx/readme.md]: https://docs.travis-ci.com/user/migrate/open-source-repository-migration

Comment (ChristianMurphy) [reply packages/create-mdx/readme.md]: It took a couple toggles to get it right, but the migration appears to have stuck this time.

Comment (ChristianMurphy) [reply packages/create-mdx/readme.md]: Confirmed, https://github.com/mdx-js/mdx/pull/599/checks is now populated. Which only the new Travis CI integration does.

Comment (wooorm) [reply packages/create-mdx/readme.md]: wow that’s cool!

**Pull request merged**